internal/promotion/runner/builtin/schemas/git-wait-for-pr-config.json

```
@@ -11,8 +11,8 @@
     },
     "provider": {
       "type": "string",
-      "description": "The name of the Git provider to use. Currently only 'github', 'gitlab', 'gitea' and 'azure' are supported. Kargo will try to infer the provider if it is not explicitly specified.",
-      "enum": ["github", "gitlab", "azure", "gitea"]
+      "description": "The name of the Git provider to use. Currently 'github', 'gitlab', 'bitbucket', 'gitea' and 'azure' are supported. Kargo will try to infer the provider if it is not explicitly specified.",
+      "enum": ["github", "gitlab", "azure", "gitea", "bitbucket"]
```

As this list continues growing, it's probably time that we alphabetize it.

Also have a small preference for an Oxford comma. Not sure why it was already missing from here, but if we're touching this line, it's a good time to fix it.

```suggestion
      "description": "The name of the Git provider to use. Currently 'azure', 'bitbucket', 'gitea', 'github', and 'gitlab' are supported. Kargo will try to infer the provider if it is not explicitly specified.",
      "enum": ["azure", "bitbucket", "gitea", "github", "gitlab"]
```

Note that codegen needs to be re-run after this change.